notebooks/OpenScale.ipynb

```diff
@@ -1437,20 +1437,13 @@
     "  * Download, configure and schedule the [model feed notebook](https://raw.githubusercontent.com/emartensibm/german-credit/master/german_credit_scoring_feed.ipynb). This notebook can be set up with your WML credentials, and scheduled to provide a consistent flow of scoring requests to your model, which will appear in your OpenScale monitors.\n",
     "  * Re-run this notebook. Running this notebook from the beginning will delete and re-create the model and deployment, and re-create the historical data. Please note that the payload and measurement logs for the previous deployment will continue to be stored in your datamart, and cal be deleted if necessary."
    ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.5 with Spark 2.1",
-   "language": "python",
-   "name": "python3-spark21"
+   "display_name": "Python 3.6 with Spark",
```

Comment:
your readme is specifically asking for 3.5, but looks like you bumped the notebook to 3.6

Reply:
I'll change in README